people_tracking_v2/SegmentedImages segmented_images
---
people_tracking_v2/HoCVector[] results
